Bdd/features/environment.py

```
@@ -88,6 +140,18 @@ def after_scenario(context, scenario):
             logger.warning("Failed to restore syslog-ng config in teardown: %s", exc)
         context.syslog_ng_config_changed = False
 
+    # Restart the OTel oracle if a "stops accepting TCP" step killed it during
+    # this scenario. taskkill takes down all four listeners (UDP/TCP/TLS/mTLS)
+    # and `Given the syslog oracle is running` only records counts — it does
+    # not start anything — so without this every subsequent scenario fails.
+    if (getattr(context, "oracle_format", None) == "otel-jsonl"
+            and getattr(context, "otel_oracle_paused", False)):
+        try:
+            otel_start_oracle()
+        except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)